app/models/github_repository.rb

```diff
@@ -22,6 +22,12 @@ def get_starter_code_from(source)
     end
   end
 
+  def import_progress(**options)
```

Comment:
This is awesome, super happy we can do this through the api 🙌 

It's still an API call each time though, so we should try to call as few times as we can.